src/prompt/files_prompt.rs

```diff
@@ -97,14 +97,16 @@ impl<'a> FilesPrompt<'a> {
                     }
                 }
                 Some((KeyCode::Enter, _, _, _)) => {
-                    let selected = self
+                    let selected: Vec<String> = self
```

Comment:
Seems a bit silly but the type cannot be inferred for whatever reason so manual typing was necessary.